### ISSUE TITLE
Throttle throws division by zero

### ISSUE THREAD

**Issue by drewhk** [akka/akka#19862]
```
[akka://default/user/StreamSupervisor-0/flow-1-0-unknown-operation] Error in stage [Throttle]: / by zero
java.lang.ArithmeticException: / by zero
at akka.stream.impl.Throttle$$anon$1$$anon$2.onPush(Throttle.scala:57)
```

**Comment by drewhk** [comment]
Also, throttle started from (effectively) zero tokens in the bucket

**Comment by agolubev** [comment]
@drewhk the problem here is that we can do throttling for up to one second. Because now we have multiplier 2^30 in case we need to support 24h it must be 2^46.
So the question is what minimal speed throttle should support? 1 message in 1h?

**Comment by drewhk** [comment]
There is a fix coming I kust need to finish the PR
2016. márc. 12. 20:32 ezt írta ("Alexander Golubev" <
notifications@github.com>):

**Comment by pirkkajokela** [comment]
I just ran in to this. Is there some set of parameters what work with throttling? (In my use, the exact pace is not that critical, so I could fudge the params if that helps.)

**Comment by drewhk** [comment]
No, this cannot be fixed by parameter fudging. The fix will be in 2.4.3, due next Friday.
